kafka-impl/src/main/java/io/streamnative/pulsar/handlers/kop/EndPoint.java

```diff
@@ -30,6 +30,7 @@
  */
 public class EndPoint {
 
+    @Getter
```

Comment:
For a `public static final` field, the getter is useless and looks weird. It's better to just modify it to public.

Reply:
good idea